modules/candidate_parameters/jsx/monitoring.js

```diff
@@ -0,0 +1,449 @@
+/* exported RMonitoringStatus */
+
+var MonitoringStatus = React.createClass(
```

Comment:
Overall would be nice to use ES6 syntax, or at least get rid of all `vars` in favour of `let` and `const`